packages/hashi/test/kolibri.spec.js

```diff
@@ -0,0 +1,89 @@
+// import flatMap from 'lodash/flatMap';
+import Mediator from '../src/mediator';
+import Kolibri from '../src/kolibri';
+
+describe('the kolibri hashi shim', () => {
+  let kolibri;
+  let mediator;
+  let response;
+  beforeEach(() => {
+    mediator = new Mediator(window);
+    kolibri = new Kolibri(mediator);
+    kolibri.__setShimInterface();
+  });
+  describe('__setShimInterface method', () => {
+    it('should set kolibri shim property', () => {
+      expect(kolibri.shim).not.toBeUndefined();
+    });
+  });
+  describe('iframeInitialize method', () => {
+    it('should set a "kolibri" property on object', () => {
+      const obj = {};
+      kolibri.iframeInitialize(obj);
+      expect(obj.kolibri).toEqual(kolibri.shim);
+    });
+  });
+
+  describe('getContentById method', () => {
+    beforeEach(function() {
+      response = { node: { id: 'abc123' } };
+      kolibri.shim.getContentById = jest.fn().mockResolvedValue(response);
+    });
+    it('should be called once', () => {
+      kolibri.shim.getContentById().then(() => {
+        expect(kolibri.shim.getContentById).toHaveBeenCalledTimes(1);
+      });
+    });
+    it('should return a promise that resolves to a ContentNode object', () => {
+      return kolibri.shim.getContentById().then(data => {
+        expect(data).toEqual(response);
+      });
+    });
+  });
+
+  describe('getContentByFilter method', () => {
+    beforeEach(function() {
+      response = [{ 1: [{ node: 'abc' }] }, { 2: [{ node: '123' }] }];
```

Comment:
Should the `response` match the `typedef` for `PageResult` e.g. `{page, pageSize, results}`? Or does it get transformed inside the shim?